lib/web/fotorama/fotorama.js

```diff
@@ -1140,7 +1153,12 @@ fotoramaVersion = '4.6.4';
 
     function addEvent(el, e, fn, bool) {
         if (!e) return;
-        el.addEventListener ? el.addEventListener(e, fn, !!bool) : el.attachEvent('on' + e, fn);
+
+        const options = (Modernizr.passiveeventlisteners) ? {
+            passive: e === 'wheel'
```

Comment:
We seem to have lost the setting for `capture` here.
```suggestion
            passive: e === 'wheel',
            capture: !!bool,
```

Reply:
Thanks for your review! Added second param capture

Reply:
I can see you've reversed this decision in fa7a9a4bf7733c9c5340680a207811a4d4f3f753. Can you provide some information to explain why?